packages/shift-react-components/src/components/navigation/navbar.js

```diff
@@ -1,6 +1,10 @@
 // Libraries
 import React, { Component } from 'react'
 import t from 'typy'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import isEqual from 'lodash/isEqual'
```

Comment:
Do we need to use lodash here?

Reply:
Just looked at alternatives to lodash, not much available. Seems you're probably going about this in the most performant way. So forget my previous comment, i'm happy with this.

Reply:
No, but I didn't want to write my own deep equality checker ;)